_episodes_rmd/13-dplyr.Rmd

```diff
@@ -50,9 +50,9 @@ even find the `dplyr` grammar easier to read.
 > ## Tip: Tidyverse
 >
 > `dplyr` package belongs to a broader family of opinionated R packages
-> specifically designed for data science called "Tidyverse". Some of these
-> packages will be covered along this course, but you can find more 
-> complete information here: https://www.tidyverse.org/.
+> specifically designed to work harmoinously for data science called the "Tidyverse".
```

Comment:
is this meant to be "to work harmoniously"

Reply:
Yes, thank you for pointing out the typo! I fixed the error and made a new commit to the branch.